src/doc/rustdoc/src/lints.md

```diff
@@ -274,7 +278,11 @@ warning: unopened HTML tag `script`
 2 | | /// </script>
   | |_____________^
   |
-  = note: `#[warn(invalid_html_tags)]` on by default
+  note: the lint level is defined here
+ --> foo.rs:1:9
+  |
+1 | #![warn(rustdoc::invalid_html_tags)]
+  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Note based on discussion with @jyn514 on Zulip: despite the output change here, the lint was not previously enabled by default. The old output just wasn't accurate for the current state of rustc.

I've filed https://github.com/rust-lang/rust/issues/81305 about getting these files checked as doctests. I don't consider that a blocker for this PR, though.